megablocks/backend/kernels.py

```diff
@@ -262,8 +243,8 @@ def _padded_copy_wgrad(
     x += tl.multiple_of(index_x * NUM_COLUMNS, NUM_COLUMNS)
     offsets = tl.max_contiguous(tl.arange(0, BLOCK_X), BLOCK_X)
 
-    acc = tl.zeros((BLOCK_X,), dtype=tl.float32)
-    iterations = tl.cdiv(NUM_COLUMNS, BLOCK_X)
+    acc = tl.zeros((BLOCK_X, ), dtype=tl.float32)
+    # iterations = tl.cdiv(NUM_COLUMNS, BLOCK_X)
```

Comment:
Why is this commented out?

Reply:
This was an unused parameter and rather than deleting it, I just commented out. I flagged this above.

I think I'm going to delete all of these unused parameters and if things break, I'll put them back.